### PR TITLE
feat(cloudcontrolspartner): generate library

### PR DESCRIPTION
**Pull request by scotthart**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13744)

### CODE REVIEW

File: google/cloud/cloudcontrolspartner/CMakeLists.txt [hunk 1]
```diff
@@ -31,6 +31,8 @@ if (BUILD_TESTING AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
             $<TARGET_FILE:cloudcontrolspartner_quickstart> GOOGLE_CLOUD_PROJECT
             GOOGLE_CLOUD_CPP_TEST_REGION # EDIT HERE
     )
-    set_tests_properties(cloudcontrolspartner_quickstart
-                         PROPERTIES LABELS "integration-test;quickstart")
+    set_tests_properties(
+        cloudcontrolspartner_quickstart
+        PROPERTIES LABELS "integration-test;quickstart" PASS_REGULAR_EXPRESSION
+                   "Permanent error.*gcloud-cpp.retry.function=ListCustomers")
```
Comment (devbww) [inline google/cloud/cloudcontrolspartner/CMakeLists.txt:37]: :-(

File: google/cloud/cloudcontrolspartner/v1/.repo-metadata.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "language": "cpp",
     "library_type": "GAPIC_AUTO",
     "name_pretty": "Cloud Controls Partner API",
-    "product_documentation": "https://cloud.google.com/sovereign-controls-by-partners/docs/sovereign-partners/reference/rest",
+    "product_documentation": "https://cloud.google.com/sovereign-controls-by-partners/docs/",
```
Comment (devbww) [inline google/cloud/cloudcontrolspartner/v1/.repo-metadata.json:10]: It looks like `generate-libraries` doesn't like that edit.

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13744)* status: 0 of 83 files reviewed, 2 unresolved discussions (waiting on @devbww)

___
*[`google/cloud/cloudcontrolspartner/CMakeLists.txt` line 37 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13744#gh-1516852328:-NsPsTXj525g2MNJzI57:b-aealcb) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/c3a8e1eade7f4c997d07e0643834143640f2b86e/google/cloud/cloudcontrolspartner/CMakeLists.txt#L37)):*
<details><summary><i>Previously, devbww (Bradley White) wrote…</i></summary><blockquote>

:-(
</blockquote></details>

Not sure this can be helped for this particular service.
___
*[`google/cloud/cloudcontrolspartner/v1/.repo-metadata.json` line 10 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13744#gh-1516860452:-NsPsYlf6B_O2y4WrDU6:b-tqo50h) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/c3a8e1eade7f4c997d07e0643834143640f2b86e/google/cloud/cloudcontrolspartner/v1/.repo-metadata.json#L10)):*
<details><summary><i>Previously, devbww (Bradley White) wrote…</i></summary><blockquote>

It looks like `generate-libraries` doesn't like that edit.
</blockquote></details>

Removed edit.

**Review by devbww — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/13744)* status: 0 of 83 files reviewed, all discussions resolved

**Pull request merged**